src/cli.py

```diff
@@ -42,11 +42,14 @@ def run(lat=0, long=0, args=None):
     # Makes calls to the apis(ocean, UV) and returns the values in a dictionary
     ocean_data_dict = api.gather_data(lat, long, arguments)
 
+    # Sets hourly_dict = the dictionary for hourly weather data
+    hourly_forecast = api.get_hourly_forecast(lat, long, days=1)
+
     # Non-JSON output
     if arguments["json_output"] == 0:
         # Response prints all the outputs & returns the GPT response
         response = helper.print_outputs(
-            city, ocean_data_dict, arguments, gpt_prompt, gpt_info
+            hourly_forecast, ocean_data_dict, arguments, gpt_prompt, gpt_info
```

Comment:
the `ocean_data_dict` should have all the hourly forecast data, if im not mistaken. It doesn't seem necessary to pass it as an argument